From: SeongJae Park <sj@kernel.org>
Date: Sun, 13 Jul 2025 13:47:24 -0700
Subject: [PATCH] ==== more selftest ====

selftests/damon/sysfs.py: test all parameters

sysfs.py tests if DAMON sysfs interface is passing the user-requested
parameters to DAMON as expected.  But only the default (minimum)
parameters are being tested.  This is partially because _damon_sysfs.py,
which is the library for making the parametr requests, is not supporting
the entire parameters.  Internal DAMON status dump script
(drgn_dump_damon_status.py) is also not dumping entire parameters.
Extend the test coverage by updating parameters input and status dumping
scripts to support all parameters, and writing additional tests using
those.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_75q62vqb | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_75q62vqb

diff --git a/hkml_cv_bogus/hkml_cv_bogus_75q62vqb b/hkml_cv_bogus/hkml_cv_bogus_75q62vqb
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.5

